apps/web/src/lang/fr.json

```diff
@@ -3423,6 +3423,10 @@
     "defaultMessage": "Créer une équipe ",
     "description": "Link text to create a new team in the admin portal"
   },
+  "H+83NU": {
+    "defaultMessage": "Vous avez des questions? <link>Contactez-nous! </link>.",
```

Comment:
Thanks for catching this.  There were a few more weird spacings I needed to fix as well.
Ref: [fix some translations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8544/commits/b0cf494dc2350c6c07d75b1ef8d77d7b7be34c8d)

Reply:
Related to translations, our date library uses a colon for the French locale but our translates are using the h prefix.  I'm not sure if this is a problem or not.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/9af890e2-d43d-4aec-9483-81e810e39be6)

Reply:
Oof, this should be consistent, but I can't provide insight into which direction is preferable.

Reply:
According to http://www.noslangues-ourlanguages.gc.ca/en/cles-de-la-redaction/heure-ecriture-de-lheure.html the colon can be used in some situations.  Since I'd like to keep leaning on our formatting library and not start writing our own, I'm voting for colons. :grin: 
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/1f14c18a-5b12-44ac-9032-f747d4d1518d)

Reply:
[change time to colon](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8544/commits/0f0bbe4276b355c1b21d4b61052eacf42a49ec89)
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a1f97496-d002-491b-9f88-5b64ca70185f)

Reply:
Is this a French thing or should I trim the space before the comma?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/672f821c-e889-46b8-b0db-23ffa03daf62)

Reply:
I usually lean on @mnigh for this type of question but @gobyrne can likely help too!

Reply:
Aye aye, [trim the space](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8544/commits/d72bcd09009e75b46e143bdcfe256efc12df6b47)